_main.cfg

```
@@ -58,17 +58,19 @@ data/add-ons/Flight_Freedom_Resources/music#enddef
     {ENABLE_DWARVISH_RUNESMITH}
     {ENABLE_TROLL_SHAMAN}
     {ENABLE_SAURIAN_SPEARTHROWER}
+    {ENABLE_NIGHTBLADE}
+    {ENABLE_PARAGON}
 
     [modify_unit_type]
         type=Drake Clasher
         add_advancement=Drake Guard
-        set_experience=41
+        #set_experience=41
```

why comment this out?

---

since set_experience isn't a mandatory key for modify_unit_type; if a future Wesnoth version changes the exp needed for advancement it doesn't need to be overridden.

---

I think it might be better to explicitly override it anyways, for the sake of stability and consistency across versions. If a future Wesnoth version changes the exp needed for advancement, and we *don't* override that amount, then we'd need to do some other sort of rebalancing instead...

---

sure but I feel that could go either way, i.e. if a future Wesnoth version changes the exp and adjusts some other value to compensate, etc.

---

Well ok, could we at least just switch it to ifdef-ing it out instead of commenting it out, then? I'm thinking like:
```suggestion
#ifdef FORCE_EXP_AMT
        set_experience=41
#endif
```

---

Updated!

---

ok, I see that was in f398700...